dts/bindings/sensor/adi,adltc2990.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2023 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: ADLTC2990 Quad I2C Voltage, Current and Temperature Monitor
+
+compatible: "adi,adltc2990"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  reg:
+    required: true
+    description: |
+      i2c device address
+
+  mode_v1_v2:
+    type: int
+    required: true
+    description: |
+      Define how these channels should be used
+      nothing = 0
+      current = 1
+      differential = 2
+      single-ended = 3
+      temperature = 4
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+      - 4
+
+  pins_v1_v2_current_resistor:
+    type: int
+    description: Define the resistor to be used for measuring current in micro ohms
+    default: 1000
+
+  pin_v1_voltage_divider_resistors:
+    type: array
+    description: Define the resistor to be used for measuring Vout in milliohms
+    default: [100, 10]
+
+  pin_v2_voltage_divider_resistors:
+    type: array
+    description: Define the resistor to be used for measuring Vout in milliohms
+    default: [100, 10]
+
+  mode_v3_v4:
+    type: int
+    required: true
+    description: |
+      Define how these channels should be used
+      nothing = 0
+      current = 1
+      differential = 2
+      single-ended = 3
+      temperature = 4
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+      - 4
```

Comment:
Please look at following the linux kernel bindings https://www.kernel.org/doc/Documentation/devicetree/bindings/hwmon/ltc2990.txt

Reply:
Hi @MaureenHelm thanks for the detailed review, That was also my initial idea to have a combination of mode_4_3 and mode_2_0 control register values. However, I thought of simplifying it for the user to select just the physical value to be measured. In the approach from linux kernel bindings, a deeper understanding of the combination of the control register values is required. Would it be okay to leave the mode selection like this?